src/TikiClient/index.ts

```diff
@@ -235,13 +242,17 @@ export default class TikiClient {
       licenseDesc: "",
       expiry: undefined,
       titleDesc: undefined,
-      terms: await instance.license.terms(
-        instance.config.companyName,
-        instance.config.companyJurisdiction,
-        instance.config.tosUrl,
-        instance.config.privacyUrl
-      ),
+      terms: terms
     };
+
+
+    const licenseSignature = await Utils.signMessage(
+      JSON.stringify(licenseReq),
```

Comment:
We need to remove the user_signature from the licenseRequest before signing, to prevent the `user_signature: ""`value in json.

Reply:
this doesn't happen